test/project_types/script/layers/infrastructure/script_service_test.rb

```diff
@@ -447,6 +447,16 @@
     end
   end
 
+  describe 'when shop domain is formatted' do
+    it 'should remove the ending /' do
+      assert_equal('shop.myshopify.com', script_service.send(:format_shop_domain, 'shop.myshopify.com/'))
+    end
+
+    it 'should not be modified' do
+      assert_equal('shop.myshopify.com', script_service.send(:format_shop_domain, 'shop.myshopify.com'))
+    end
+  end
+
```

Comment:
We shouldn't unit test private methods this way.  We should update the `enable()` and `disable()` to include test cases where shop domains is with and without `/`.  Ultimately, regardless this private method is correct, if `enable()` erroneously didn't call it, that would be bad.